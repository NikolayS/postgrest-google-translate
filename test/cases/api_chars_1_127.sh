#!/bin/bash

psql -U postgres test -c "select v1.google_translate('ru', 'en', string_agg(char, '')) from (select chr(generate_series) as char from generate_series(1, 126)) as z"
