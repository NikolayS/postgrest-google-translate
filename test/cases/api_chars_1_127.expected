                                                          google_translate                                                          
------------------------------------------------------------------------------------------------------------------------------------
 !. &quot;# $% &amp; &#39;() * +, - / 0123456789:; &lt;=&gt; @ ABCDEFGHIJKLMNOPQRSTUVWXYZ [\\] ^ _` abcdefghijklmnopqrstuvwxyz | ~?
(1 row)

